contracts/cosmwasm-vm/cw-xcall/src/send_call_message.rs

```diff
@@ -89,13 +99,13 @@ impl<'a> CwCallService<'a> {
             .add_attribute("method", "send_packet")
             .add_attribute("sequence_no", sequence_no.to_string())
             .add_event(event);
-        if protocol_fee > 0 {
```

Comment:
This if check might still be needed?

Reply:
since we have already asserted total paid to be greater . i think it would be non negative right?

Reply:
The check might be relevant when no fees has been set. Hence updated.